cfpb/cfgov-refresh
# Update for capital-framework 5.1.0

Updates the project to use the latest release from Capital Framework.

## Changes

- Updates cf-icons to 4.4.0, adding the updating alias to the update icon

## Testing

- Run `npm install`
- Run `gulp clean && gulp build`
- Run `./runserver.sh`
- Create a page and add a full-width text, then a heading, set the icon to `updating`
- Preview the page, you should see the new animated updating icon (it should spin)

## Screenshots


## Notes

- I can't find where this icon is currently used, if anyone knows of a place I should replace it, please reply below.

## Todos

- Search the repo to ensure we've removed all of the old icon fonts and then remove the icon font from the build process.

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
